pyof/foundation/network_types.py

```diff
@@ -366,8 +366,8 @@ def __init__(self, version=4, ihl=5, dscp=0, ecn=0, length=0, # noqa
         self.checksum = checksum
         self.source = source
         self.destination = destination
-        self.options = options
-        self.data = data
+        self.options = BinaryData() if options is None else options
+        self.data = BinaryData() if data is None else data
```

Comment:
`BinaryData()` -> `b''`; it's bad having GenericTypes when they are not needed.
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.
(for both lines)